src/frontend/src/handler/create_mv.rs

```diff
@@ -176,6 +182,36 @@ pub async fn handle_create_mv(
     .await
 }
 
+/// Send a provision request to the serverless backfill controller
+pub async fn provision_serverless_backfill(sbc_addr: String) -> Result<String> {
```

Comment:
The name `provision_serverless_backfill` is a bit confusing. `provision_resource_group` is better.